sdk/metrics/sdk.go

```diff
@@ -0,0 +1,386 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"bytes"
+	"context"
+	"sort"
+	"sync"
+	"sync/atomic"
+	"unsafe"
+
+	"go.opentelemetry.io/api/core"
+	api "go.opentelemetry.io/api/metric"
+	"go.opentelemetry.io/sdk/export"
+)
+
+// TODO: support disabled metrics at the SDK level
```

Comment:
I think we dropped the "disabled" thing from instruments in the spec, right? If so, then this is outdated.